libwnbd/libwnbd.vcxproj

```diff
@@ -19,14 +19,12 @@
     <ClCompile Include="utils.cpp" />
     <ClCompile Include="wnbd_ioctl.cpp" />
     <ClCompile Include="wnbd_log.c" />
```

Comment:
wnbd_wmi.cpp[h] needs to be remove from libwnbd/libwnbd.vcxproj.filters as well

Reply:
Good point, thanks!